docs/Makefile

```diff
@@ -4,7 +4,7 @@ HUGO_IMG?=hugomods/hugo:0.115.3
 
 THEME_MODULE = github.com/nginxinc/nginx-hugo-theme
 ## Pulls the current theme version from the Netlify settings
```

Comment:
Is this still correct?

Reply:
No, but fixing it is outside the scope of the PR.

The Makefile and README are due an update, which will be handled  separately.